spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -89,6 +89,8 @@ However, some Java types are not legal bean types :
 * A parameterized type that contains a wildcard type parameter is not a legal bean type.
 * An array type whose component type is not a legal bean type.
 
+If the set of bean types for a bean contains non-legal bean types, the container will ignore them during <<typesafe_resolution, typesafe resolution>>.
```

Comment:
I hesitated to put it in each part. If we go that way (putting in different part) we should remove it from here to avoid denormalisation of the spec. wdyt ?